### ISSUE TITLE
VerdorExtension support type of boolean

### ISSUE THREAD

**Issue by liubao68** [springfox/springfox#3328]
Please take the time to search the repository, if your question has already been asked or answered.

- [x] What version of the library are you using?  the latest version

What kind of issue is this?

 - [x] Feature Request. Start by telling us what problem you’re trying to solve. Often a solution
       already exists! Don’t send pull requests to implement new features without first getting our
       support. Sometimes we leave features out on purpose to keep the project small.

I checked VendorExtensionsMapper , there are only three types fo VerdorExtensions support. However, I want to create a boolean value as follows

```yaml
  /stringRequestBody:
    post:
      tags:
        - "schema-content-controller"
      summary: "stringRequestBody"
      operationId: "stringRequestBodyUsingPOST"
      schemes: []
      consumes:
        - "application/json"
      produces:
        - "application/json"
      parameters:
        - in: "body"
          name: "id"
          description: "id"
          required: true
          schema:
            type: "string"
          x-raw-json: true
```

Add 

```
x-raw-json: true
```

not 

```
x-raw-json: "true"
```

not

```
x-raw-json: 
  - true
```

**Comment by liubao68** [comment]
seems fixed in https://github.com/springfox/springfox/commit/4f500fc7bd7853c37febb1e4858704a3ca7ee8a3#diff-18d578563a8072f764d311545d902f86

but no version available.

**Comment by dilipkrish** [comment]
Its available in 2.10.5, but 3.0.0 should be released shortly

**Comment by liubao68** [comment]
Thank you, not found 2.10.5 in [relase](https://github.com/springfox/springfox/releases)  but find 2.10.5 in tags.
